AzureDevOps/DistributeTests.ps1

```diff
@@ -0,0 +1,44 @@
+<#  
```

Comment:
Hi @Vahila, are the comments in this file for external users? I am asking this due to comments such as "**We** search ..."

Reply:
Hi @mw-hrastega, yes these are intended as reference for users to get started.